### DISCUSSION TITLE
3월 4주 리팩터링 내역 (230321 ~ 230331)

### DISCUSSION THREAD

**Discussion by punchdrunkard** [dnd-side-project/dnd-8th-1-frontend#318] [01. Refactoring]
리팩터링 작업 관련 내용을 작성해주세요!

**Comment by punchdrunkard** [comment]
위키 고칠 부분
- Tech stack에 버전 명시 (위키에선 빼고 리드미에 있는 부분이긴함) , 표로 작성 => 위키에 우선 적어둡시다. 
- 제가 완성 못시킨 부분

**Comment by punchdrunkard** [comment]
README에 들어갈 부분 정하기

**Reply by punchdrunkard** [reply]
노션 참고

**Reply by jinyoung234** [reply]
리드미 작성완료!(진영 담당)

**Comment by punchdrunkard** [comment]
버그 고치기

**Reply by punchdrunkard** [reply]
meet 메인 페이지에서 페이지 이동 후 필터 선택하면 에러뜸

<img width="324" alt="image" src="https://user-images.githubusercontent.com/74234333/226614405-df7e2af4-2154-44c9-821d-6177fadab007.png">

**Reply by punchdrunkard** [reply]
만나기 작성 페이지 title 설정

**Reply by punchdrunkard** [reply]
만나기 작성 페이지 로그인하지 않으면 접근 제어

**Reply by punchdrunkard** [reply]
공연 글 등록할 때 프로필 등록하지 않으면 에러 발생 함 (진영)

**Reply by punchdrunkard** [reply]
만나기 글 등록할 때 프로필 등록하지 않으면 에러 발생 함 (진영)

**Comment by punchdrunkard** [comment]
폴더 구조 개선

```
components
 - base
 - domain
    - main
    - meet
      - edit
      - detail
    - performance
    - profile
    - signin
```

**Reply by punchdrunkard** [reply]
atoms, molecules => 다은
organisms, templates => 진영

**Reply by punchdrunkard** [reply]
스토리북까지 해야함

**Reply by jinyoung234** [reply]
진영 담당 완료,,
